Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -314,22 +320,31 @@ - (void)setPlaceholderTintColor:(UIColor *)placeholderTintColor
 - (void)setSelected:(BOOL)selected
 {
     [super setSelected:selected];
-    if (self.isSelected) {
-    } else {
-        self.positionLabel.hidden = YES;
-    }
+    [self updatePositionLabelToSelectedState:self.isSelected];
 }
 
 - (void)tintColorDidChange
 {
     [super tintColorDidChange];
     _selectionFrame.layer.borderColor = [[self tintColor] CGColor];
-    _positionLabel.backgroundColor = [self tintColor];
-    if (self.isSelected) {
 
+    [self updatePositionLabelToSelectedState:self.isSelected];
+}
+
+- (void)updatePositionLabelToSelectedState:(BOOL)selected
+{
+    if (selected) {
+        _positionLabel.backgroundColor = [self tintColor];
+        _positionLabel.layer.borderColor = [self tintColor].CGColor;
+        _positionLabel.layer.shadowColor = [UIColor colorWithWhite:0 alpha:1].CGColor;
```

Comment:
I need to remove that shadowColor just realize that shadow is applied for the text on the label , and not for the border/circle.